README.md

```diff
@@ -74,6 +74,9 @@ The consuming service makes a POST request to `/upscan/initiate` or `upscan/v2/i
 This request must contain a `User-Agent` header that can be used to identify the service, but an allow list of authorised services is no longer cross-checked.
 The service must also provide a callbackUrl for asynchronous notification of the outcome of an upload. The callback will be made from inside the MDTP environment. Hence, the callback URL should comprise the MDTP internal callback address and not the public domain address.
 
+**Note:** Although a `User-Agent` header is always required, consuming services may wish to explicitly identify themselves by including a `"consumingService"` field in the request body. 
+This functionality may be useful to you if the actual service that initiates the request is different to your preferred logical service.
+
```

Comment:
Would it be worth mentioning something about the user-agent needing to be present in the upscan-app-config (or the default Mime Types will be allowed)? It might be too much info so at your discretion